contracts/ts/genMaciState.ts

```diff
@@ -237,12 +235,6 @@ export const genMaciStateFromContract = async (
         break;
       }
 
-      // ensure that the message root is correct (i.e. all messages have been published offchain)
-      case action.type === "MergeMessageAq": {
```

Comment:
we are still pulling events for mergeMessageSubroots and mergeMessageAq, let's remove those

Reply:
resolved